src/controllers/enter.information.controller.ts

```diff
@@ -11,7 +11,7 @@ import {
   retrieveFromObjectionSession,
 } from "../services/objection.session.service";
 
-export const get = async (req: Request, res: Response, next: NextFunction) => {
+export const get = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Unrelated to company number validation but noted that there was a lint failure due to this.

Reply:
Can we keep this please, when my PR gets merged in it will need it.